INSTALL.md

```diff
@@ -125,6 +125,85 @@ source.
 
 ### Windows ###
 
+Windows installation support has recently been enhanced. The new installation
+method doesn't require Cygwin or WSL and relies only a light weight git/bash
+distribution like Git-Bash, and the Intel OneAPI base distribution as well as
+the Intel OneAPI HPC Toolkit. This allows native and redistributable statically
+linked binaries to be built if you use the appropriate compiler flags. Clients
+will only require the free redistributable Intel MPI runtime.
+
+#### New and Recommended Windows Installation ####
+
+Please follow these instructions, this is the recommended way to install
+OpenCoarrays on Windows.
+
+##### Pre-requisits #####
+
+* [CMake]
+* [Git-Bash]
```

Comment:
```suggestion
* [Git BASH]
```

Reply:
This seems inconsistent with other references to Bash within our own code-base and, indeed with other [references to Bash throughout the internet](https://www.google.com/search?q=bash&ei=Q9J1YqmZA9iHytMPlda6yAQ&ved=0ahUKEwjpxt78pMz3AhXYg3IEHRWrDkkQ4dUDCA4&uact=5&oq=bash&gs_lcp=Cgdnd3Mtd2l6EAMyBggAEAoQQzIKCAAQsQMQgwEQQzIHCAAQsQMQQzILCAAQgAQQsQMQgwEyCAgAEIAEELEDMgsIABCABBCxAxCDATIICAAQsQMQgwEyCwgAEIAEELEDEIMBMgsIABCABBCxAxCDATIICAAQgAQQsQM6BwgAEEcQsANKBAhBGABKBAhGGABQwARYyAdgsAxoAXABeACAAb8BiAHQApIBAzAuMpgBAKABAcgBCMABAQ&sclient=gws-wiz) I think I'm going to leave it as is for now. It also appears that Git-Bash has been rebranded to [Git for Windows](https://gitforwindows.org/)... so maybe we should update these references to that?